src/exts/start_new_spring_handler.js

```diff
@@ -0,0 +1,25 @@
+const { join } = require('path');
+
+const { bridge } = require('../spring_api');
+const springPlatform = require('../spring_platform');
+const launcher = require('../engine_launcher');
+
+bridge.on('StartNewSpring', async command => {
+	// Fetch the engine version
+	const enginePath = join(
+		springPlatform.writePath,
+		'engine',
+
+		// We need to put the engine's version in lowercase, because the
+		// directory name is in lowercase, and FS is case sensitive on
+		// some OSes.
+		command.Engine.toLowerCase(),
+
+		springPlatform.springBin
+	);
+
+	// Build the replay's path
```

Comment:
please remove these obvious comments:
- // Build the replay's path
- // Fetch the engine version

This one is good because it's not obvious why this operation is being done.
- // We need to put the engine's version in lowercase, because the
In general I'd like comments to be used sparingly, but with great effect.